.github/workflows/gradle.yml

```diff
@@ -0,0 +1,77 @@
+name: Web3j
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  integration-test:
+    runs-on: ubuntu-latest
+    env:
+      registry.username: ${{ secrets.REGISTRY_USERNAME }}
+      registry.password: ${{ secrets.REGISTRY_PASSWORD }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+      - name: Run integration tests
+        run: ./gradlew integrationTest
+  build:
+    runs-on: ubuntu-latest
+    env:
+      OSSRH_USERNAME: ${{ secrets.OSSRH_USERNAME }}
+      OSSRH_PASSWORD: ${{ secrets.OSSRH_PASSWORD }}
+      GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
+      BINTRAY_USER: ${{ secrets.BINTRAY_USER }}
+      BINTRAY_KEY: ${{ secrets.BINTRAY_KEY }}
+      GITHUB_PERSONAL_ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Decrypt secret key
+        run: openssl aes-256-cbc -K ${{ secrets.GPG_KEY }} -iv ${{ secrets.GPG_IV }} -in web3j.asc.enc -out web3j.asc -d
+        if: github.base_ref == ''
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+      - name: Build pull request
+        run: ./gradlew check jacocoTestReport
+        if: github.base_ref != ''
+      - name: Build master
+        run: ./gradlew check jacocoTestReport publish
+        if: github.base_ref == ''
+      - name: Build and stage release
+        run: ./gradlew check publish bintrayUpload -Pversion=${GITHUB_REF#'release/'}
+        if: contains(github.ref, '/release/')
+      - name: Release
+        run: ./gradlew closeAndReleaseRepository
+        if: contains(github.ref, '/release/')
+      - name: GitHub release
+        run: ./scripts/github-release.sh
+        if: contains(github.ref, '/release/')
+      - name: Javadoc release
+        run: ./gradlew -Dorg.ajoberstar.grgit.auth.username=${{ secrets.GITHUB_TOKEN }} core:gitPublishPush
```

Comment:
Are we releasing java docs only for `core` ? Why not release the docs for all the library ?

Reply:
Not sure about that, I think for the moment only core.

Reply:
yes we are releasing the docs only for core